plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -131,8 +131,8 @@ def upload(repo, type_id, unit_key, metadata, file_path, conduit, config):
     except PulpCodedException, e:
         _LOGGER.exception(e)
         return _fail_report(str(e))
-    except:
-        msg = 'unexpected error occurred importing uploaded file'
+    except Exception, e:
```

Comment:
It's better to use the newer ``as`` syntax, because it's compatible with python 3.

``except Exception as e:``